pkgs/applications/office/bookworm/default.nix

```diff
@@ -40,6 +40,10 @@ stdenv.mkDerivation rec {
   postPatch = ''
     chmod +x meson/post_install.py
     patchShebangs meson/post_install.py
+
+    # this changes the executable name
+    # from 'com.github.babluboy.bookworm' to 'bookworm'
+    sed -i "s/    meson.project_name(),/    'bookworm',/g" meson.build;
```

Comment:
I would rather make a symlink after installation. Other packages might depend on this executable to be available.